src/imguiclient/imguiapp.cpp

```diff
@@ -1410,66 +1404,77 @@ std::string ImGuiApp::create_time_string(std::chrono::time_point<std::chrono::sy
 
 void ImGuiApp::BackgroundWindow()
 {
-    // #TODO: Make the background dockable somehow. There is a example in ImGui Demo > Examples > Dockspace
+    // Configure flags for the background window
+    ImGuiWindowFlags window_flags = ImGuiWindowFlags_MenuBar | ImGuiWindowFlags_NoDocking | ImGuiWindowFlags_NoTitleBar
+        | ImGuiWindowFlags_NoCollapse | ImGuiWindowFlags_NoResize | ImGuiWindowFlags_NoMove
+        | ImGuiWindowFlags_NoBringToFrontOnFocus | ImGuiWindowFlags_NoNavFocus;
+
+    // Get the main viewport
+    const ImGuiViewport *viewport = ImGui::GetMainViewport();
+
+    // Set window position and size to cover the entire viewport
+    ImGui::SetNextWindowPos(viewport->WorkPos);
+    ImGui::SetNextWindowSize(viewport->WorkSize);
```

Comment:
Does this mean we no longer need the `m_windowPos` and `m_windowSize`?

Reply:
Yes, removed them from here and from imguiglfw

Reply:
You would also need to remove it from imguiwin32.

Reply:
Done